ui/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "metalk8s-platform-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.7.2",
+    "@kubernetes/client-node": "git://github.com/scality/kubernetes-client-javascript.git#browser",
```

```suggestion
    "@kubernetes/client-node": "scality/kubernetes-client-javascript.git#browser",
```

See this shorthand notation for GitHub [here](https://docs.npmjs.com/files/package.json#github-urls)